tests/test_transaction.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+import six
+
+from pynamodb.connection import Connection
+from pynamodb.connection.transactions import Transaction, TransactGet, TransactWrite
+
+if six.PY3:
+    from unittest.mock import MagicMock
+else:
+    from mock import MagicMock
+
+
+class TestTransaction:
+
+    def test_commit__not_implemented(self, mocker):
+        t = Transaction(connection=mocker.MagicMock())
+        with pytest.raises(NotImplementedError):
+            t._commit()
+
+
+class TestTransactGet:
+
+    def setup(self):
+        self.mock_model_cls = MagicMock(__name__='MockModel')
```

Comment:
Lots of mocking here! Is it possible to switch out some of these mocks for real objects so that we're testing more throughly?

Reply:
replaced some things!